### PR TITLE
Check in vscode settings with debug instructions

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Comment by shifucun** [comment]
Can you confirm if the .vscode settings are compatible with what you have locally?

**Review by chejennifer — APPROVED** [review]
this is very useful!

**Review by beets — APPROVED** [review]
thanks! i have very few settings in my vscode:

```
{
  "files.exclude": {
    "**/node_modules": true
  },
  "python.linting.pylintEnabled": true,
  "python.linting.enabled": true,
  "python.pythonPath": "/Users/cyin/google/website/.env/bin/python3",
  "go.inferGopath": false
}
```

perhaps we can skip adding settings.json? i'm afraid everyone's various settings will keep clobbering each other

File: .gitignore [hunk 1]
```diff
@@ -12,7 +12,6 @@ server/dist/**
 **/node_modules/**
 
 # Local env.
-.vscode
```
Comment (beets) [inline .gitignore:15]: we could perhaps limit this only allow launch.json only (there are other vscode files which we don't want to check in yet..)

Comment (shifucun) [reply .gitignore:15]: only adding launch.json now

**Pull request merged**